packages/stylelint-plugin/src/rules/no-invalid-use-of-tokens.ts

```diff
@@ -24,10 +24,10 @@ import { fixAlphaModificationFunctions } from "./lib/fixAlphaModificationFunctio
 import { containsEquationThatDoesntWorkWithCSSVariables } from "./no-invalid-use-of-var-tokens-in-equations"
 
 const deprecatedKaizenTokenPattern = new RegExp(
-  Array.from(deprecatedTokenReplacements.keys()).join("|")
+  `(${Array.from(deprecatedTokenReplacements.keys()).join("|")})`
 )
-function stringContainsDeprecatedKaizenToken(value: string) {
-  return deprecatedKaizenTokenPattern.test(value)
+function getDeprecatedKaizenTokenPatternMatches(value: string) {
+  return value.match(deprecatedKaizenTokenPattern)
```

Comment:
I'm curious: what was the reason for this change? Is `str.match(regex)` preferable to `regex.test(string)`?

Reply:
Oh you're capturing them. I get it now :)

This might have been more obvious if there was an explicit return type on the function. But the name did change, I was just slow to notice :)

Reply:
Made it more explicit :)